src/Tribe/PUE/Checker.php

```diff
@@ -488,47 +488,37 @@ private function get_api_message( $info ) {
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated and the license is expired
+		 *
 		 */
-		public function display_json_error() {
-			$pluginInfo       = $this->json_error;
-			$update_dismissed = $this->get_option( $this->dismiss_upgrade );
-
-			$is_dismissed = ! empty( $update_dismissed ) && in_array( $pluginInfo->version, $update_dismissed ) ? true : false;
-
-			if ( $is_dismissed ) {
-				return;
-			}
+		public function display_license_error_message() {
+			$plugin_info  = $this->json_error;
+			$license_tab = admin_url( 'wp-admin/edit.php?post_type=tribe_events&page=tribe-common' );
```

Comment:
Do we always want the `post_type` param to be in there? If TEC isn't active, `tribe_events` won't be defined. I don't think it will break anything right at this point in time, but it could be better to keep things clean/only add it when required.